packages/swapper/src/swappers/cow/cowBuildTrade/cowBuildTrade.ts

```diff
@@ -107,12 +107,14 @@ export async function cowBuildTrade(
     const trade: CowTrade<KnownChainIds.EthereumMainnet> = {
       rate,
       feeData: {
-        fee: '0', // no miner fee for CowSwap
+        fee: '0', // TODO: remove once web has been updated
+        networkFee: '0', // no miner fee for CowSwap
         chainSpecific: {
           estimatedGas: feeData.chainSpecific.gasLimit,
           gasPrice: feeData.chainSpecific.gasPrice,
         },
         tradeFee: '0', // Trade fees for buy Asset are always 0 since trade fees are subtracted from sell asset
+        buyAssetTradeFeeUsd: '0',
```

Comment:
```suggestion
        tradeFee: '0', // TODO: remove once web has been updated
        buyAssetTradeFeeUsd: '0', // Trade fees for buy Asset are always 0 since trade fees are subtracted from sell asset
```

Reply:
Actioned in https://github.com/shapeshift/lib/pull/1038/commits/deb7c95d42e6ce7accc9dd411351f123ea71c814.